ast/dml.go

```diff
@@ -2416,60 +2416,119 @@ func (n *FrameBound) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
-type SplitIndexRegionStmt struct {
+type SplitRegionStmt struct {
 	dmlNode
 
-	Table      *TableName
-	IndexName  string
+	Table     *TableName
+	IndexName model.CIStr
+
+	SplitOpt *SplitOption
+}
+
+type SplitOption struct {
+	Min        []ExprNode
+	Max        []ExprNode
```

Comment:
I think `lower` and `upper` are more suitable to describe ranges

Reply:
done.